website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -92,9 +92,10 @@ In addition, one of either `identity` or `service_principal` blocks must be spec
 
 * `disk_encryption_set_id` - (Optional) The ID of the Disk Encryption Set which should be used for the Nodes and Volumes. More information [can be found in the documentation](https://docs.microsoft.com/en-us/azure/aks/azure-disk-customer-managed-keys).
 
-* `identity` - (Optional) An `identity` block as defined below. Changing this forces a new resource to be created.
+* `identity` - (Optional) An `identity` block as defined below.
```

Comment:
I think we can put `One of either `identity` or `service_principal` must be specified.` in this message and then have the migration information in it's own separate note. It's a little different than how we normally do it but I'd like to make sure that we try and make it as clear as possible what a user needs to do when updating

Reply:
Done, thanks!